_docs-sources/pipelines/security/audit-log.md

```diff
@@ -0,0 +1,116 @@
+
+When Pipelines is run in response to a pull request event, the user that created the most recent commit on the branch will be used in the log, in addition to the pull request number assigned by GitHub. For example, if the user `SomeUserInYourOrg123` created a pull request that was given the number `123` by GitHub, the resulting session name would be `SomeUserInYourOrg123-via-GWPipelines@PR-123`.
+
+When Pipelines is run in response to a pull request being merged, the user that performed the merge (e.g., the user who clicked the `merge` button on the pull request) will be used in the log in addition the to deploy branch (e.g., `main`). For example, if the user `SomeUserInYourOrg123` merged a pull request to your deploy branch named `main` the session name would be `SomeUserInYourOrg123-via-GWPipelines@main`
+
+### What gets logged
+
+Logs are available for all operations performed in every AWS account by Gruntwork Pipelines. Gruntwork Pipelines takes advantage of AWS STS session names to clearly label all sessions with the GitHub user who requested the change and the pull request or branch that triggered the change.
+
+The `userIdentity` field in each CloudTrail event associated with API calls performed by Pipelines [actions](../overview/actions.md) will contain the session name. For example, if the GitHub user `SomeUserInYourOrg123` made a pull request that was the 123rd pull request in your repository, an associated CloudTrail event would have the following data for `userIdentity`.
+
+```
+{
+    "eventVersion": "1.09",
+    "userIdentity": {
+        "type": "AssumedRole",
+        "principalId": "xxxxxxxxxxxxxxxxxxxxx:SomeUserInYourOrg123-via-GWPipelines@PR-123",
+        "arn": "arn:aws:sts::123456789012:assumed-role/github/SomeUserInYourOrg123-via-GWPipelines@PR-123",
+        "accountId": "123456789012",
+        "accessKeyId": "xxxxxxxxxxxx",
+        "sessionContext": {
+            "sessionIssuer": {
+                "type": "Role",
+                "principalId": "xxxxxxxxxxxxxxxxxxxxx",
+                "arn": "arn:aws:iam::123456789012:role/github",
+                "accountId": "123456789012",
+                "userName": "github"
+            },
+            "webIdFederationData": {
+                "federatedProvider": "arn:aws:iam::123456789012:oidc-provider/token.actions.githubusercontent.com",
+                "attributes": {}
+            },
+            "attributes": {
+                "creationDate": "2023-09-25T13:24:17Z",
+                "mfaAuthenticated": "false"
+            }
+        }
+    },
+    ... rest of the CloudTrail event here ...
+}
+```
+
+Combined with a [query service](#querying-data), you can use data from CloudTrail to perform analytics on usage of Gruntwork Pipelines in your AWS accounts.
+
+## Where you can find logs
```

Comment:
This section is really well-written, but it should probably be part of our Landing Zone docs! After all, it's all general to accessing and using logs generated by CloudTrail. Could you create a separate page in that section with these guidelines?

Then this section becomes nice and short and basically says:
- We log all actions using CloudTrail
- We do specially things with the session name
- How you access CloudTrail logs depends on your org settings, but for Gruntwork Landing Zone customers, click here to see how to do it.

Reply:
Addressed in [e1ebc7c](https://github.com/gruntwork-io/docs/pull/1143/commits/e1ebc7cf969e26c9018f2f6cad23b8323e05979e)